### ISSUE TITLE
major issue with modifiers getting stuck on "Right Control + Alt Gr"

### ISSUE THREAD

**Issue by ohMuzzy** [boppreh/keyboard#442]
Similiar issue has been reported by multiple users, but I'm creating a new one, as the one described [here](https://github.com/boppreh/keyboard/issues/326) is so vague that I'm not even sure if it refers to the same situation: 

It happens when you register a hotkey (with suppress=True) that includes CTRL or ALT, and then (intentionally or probably more often accidentally) press the RIGHT CTRL + ALT GR combination.

Here it is recreated. Just run it and press RIGHT CTRL + ALT GR combination, if you have ALT GR on your keyboard. (You will need to physically press LEFT CTRL a couple of times to restore normal keyboard operation afterwards.)

```
import keyboard

def on_hotkey(str):
    print(str)

keyboard.add_hotkey('ctrl+a', on_hotkey, args=['!'], suppress=True)

keyboard.wait()
```

Pressing RIGHT CTRL + ALT GR in this example creates the following sequence of events:

**LEFT** CTRL is pressed > ALT GR is pressed > ALT GR is released > **RIGHT** CTRL is released

leaving LEFT CTRL in permanently pressed position that cannot be released virtually (only by physically pressing the key a couple of times)

There is an analogous situation when you register an ALT-containing hotkey with LEFT ALT getting stuck.

**Comment by richiehowelll** [comment]
@MuzzyGH Has anyone figured out a solution to this? I've tried adding delays, etc. and seem to run into this every so often.

**Comment by MuzzyGH** [comment]
@richiehowelll Not really, @boppreh does not seem to mantain this project anymore. I ended up coding my own solution, as every major keyboard module comes with its own flaws.

**Comment by richiehowelll** [comment]
> @richiehowelll Not really, @boppreh does not seem to mantain this project anymore. I ended up coding my own solution, as every major keyboard module comes with its own flaws.

@MuzzyGH Let me know if you ever decide to make the code for your solution available, would love to find something that just works!
